core/ir/opnd_shared.c

```diff
@@ -365,7 +365,6 @@ opnd_create_immed_int64(int64 i, opnd_size_t size)
 {
     opnd_t opnd;
     opnd.kind = IMMED_INTEGER_kind;
-    IF_X64(CLIENT_ASSERT(false, "32-bit only"));
```

Comment:
But if we enable this for 64-bit builds it gets confusing as to why int64 would be used vs int.  There is a comment in the user-facing docs in opnd_api.h that says this is 32-bit only: that should match this assert.  But see above: do we need to use this?  This PR is for AArch64, for which opnd_create_immed_int() is pointer-sized: so are you thinking of re-using some of this code for AArch32 or something and that's why this _int64 is being used?  I'm confused.

Reply:
Gah, of course! Not your confusion, mine. Reverted change and now using int operand not int64 in `cmode` functions.